boards.txt

```diff
@@ -442,6 +442,12 @@ BLENano.menu.softdevice.s130.upload.maximum_size=151552
 BLENano.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 BLENano.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_{build.chip}.ld
 
+BLENano.menu.lfclk.lfxo=Crystal Oscillator
+BLENano.menu.lfclk.lfxo.build.lfclk_flags=-DUSE_LFXO
+BLENano.menu.lfclk.lfrc=RC Oscillator
+BLENano.menu.lfclk.lfrc.build.lfclk_flags=-DUSE_LFRC
+BLENano.menu.lfclk.lfsynt=Synthesized
+BLENano.menu.lfclk.lfsynt.build.lfclk_flags=-DUSE_LFSYNT
```

Comment:
Could you please remove this menu options for now? Later on I'd like to auto select the LFCLK source depending on if the board has it build in.